surfactant/relationships/elf_relationship.py

```diff
@@ -146,6 +146,8 @@ def generate_runpaths(sw: Software, md) -> List[pathlib.PurePosixPath]:
 
 
 def replace_dst(origstr, dvar, newval) -> str:
```

Comment:
While we're adding a docstring to this function, let's also add the corresponding type hints at the same time.

```suggestion
def replace_dst(origstr: str, dvar: str, newval: str) -> str:
```